kubernetes_env/kube_env.py

```diff
@@ -237,7 +239,7 @@ def setup_bookinfo_deployment(platform, multizonal):
     result = deploy_bookinfo()
     if result != util.EXIT_SUCCESS:
         return result
-    result = deploy_addons()
```

Comment:
Why comment this out?

Reply:
They are currently not used to avoid unnecessary baggage.